tools/webdriver/webdriver/client.py

```diff
@@ -42,6 +42,11 @@ def _set(self, key, secs):
         timeouts = self.session.send_session_command("POST", "timeouts", body)
         return None
 
+    def set_w3c(self, timeout, secs):
+        body = {"type": timeout, "ms": secs * 1000}
```

Comment:
Add this into the webdriver client rather than here.

Reply:
Not sure what you mean, this is in tools/webdriver/webdriver/client.py which is the webdriver client?